frontend/src/pages/user_home.tsx

```diff
@@ -1,16 +1,25 @@
-import { Link } from 'wouter';
 import clsx from 'clsx';
-
-import { deleteDocument, useListDocuments } from '../api/document';
-import { MeButton, TopBar, TopBarPart, TopBarTitle } from '../common/top_bar';
-import { AppContainer } from '../components/app';
+import { Link } from 'wouter';
+import * as Automerge from '@automerge/automerge';
 import { AiOutlinePlus } from 'react-icons/ai';
-import { IoMdTrash } from 'react-icons/io';
-import { IconButton } from '../components/button';
+import { FaFileImport } from 'react-icons/fa';
+import { IoIosMore, IoIosTrash } from 'react-icons/io';
+import { KeyedMutator } from 'swr';
+import { deleteDocument, replaceChanges, useListDocuments } from '../api/document';
+import { MeButton, TopBar, TopBarPart, TopBarTitle } from '../common/top_bar';
 import { Version } from '../common/version';
+import { AppContainer } from '../components/app';
+import { SecondaryButton, PrimaryButton } from '../components/button';
+import { showModal, Modal } from '../components/modal';
+import { ComponentProps, useRef, useState } from 'react';
+import { DropdownItem, DropdownSection, IconDropdown } from '../components/dropdown';
+import { useDebugMode } from '../debugMode';
+
+type DocumentInfo = ReturnType<typeof useListDocuments>['data'][0];
 
 export function UserHomePage() {
   const { data, mutate } = useListDocuments({});
+  const debugMode = useDebugMode();
```

Comment:
this can be in the DocumentCard.
also: it is kinda wiered, that this is a different DebugMode than in server? Do we really need both? can we not just enable the server functionality always?

Reply:
Oh, good catch. This *should be* the backend debug mode. VSCode auto import resolution tricked me

Reply:
Regarding always enabling the server functionality: I'm strongly opposed to this, as this is a very internal tool that should not be possible during production use. And if we don't show the option in the frontend, we should disable it in the backend too.